examples/wallet/app/containers/CreateUnlockWalletPassword.js

```diff
@@ -0,0 +1,23 @@
+// @flow
```

Comment:
It's inconsistent to use the redux state to pass data between components of a single page, specially if to do so you're using a store key (in this case account) which is for something else (the account state)

What we've been doing in this app is to only use the redux store to keep 'global' information, and we try to keep the state of each page local to the components of said page. It's perfectly valid to keep *everything* in the redux state, and some projects swear by it, but it wouldn't be consistent for us now.

Daedalus, for example, has one store key for each form/page and nearly every change in the ui matches to a store action.

Reply:
Functions from parent component is used instead of redux